css/properties/column-fill.json

```diff
@@ -19,19 +19,11 @@
               },
               {
                 "prefix": "-moz-",
-                "version_added": "13",
+                "version_added": "17",
                 "version_removed": "74"
               }
             ],
-            "firefox_android": [
-              {
-                "version_added": "52"
-              },
-              {
-                "prefix": "-moz-",
-                "version_added": "14"
-              }
-            ],
```

Comment:
@ddbeck Did you investigate why the Firefox Android support was specified differently?

Reply:
Oh I misread the original data; I thought they were the same and could be mirrored without investigation. I can't test Firefox for Android this old, but [these release notes for v13](https://www.mozilla.org/en-US/firefox/android/13.0beta/releasenotes/) make me think that version 14 was an error to begin with. It probably never had a different support status from desktop. So this is a happy accident. 😄

Reply:
Thanks, makes sense.